tests/functional/test_install_wheel.py

```diff
@@ -575,6 +576,88 @@ def test_wheel_install_with_no_cache_dir(script, tmpdir, data):
     result.assert_installed('simpledist', editable=False)
 
 
+def test_wheel_install_with_ca_pool(script, tmpdir, data):
+    """Check wheel installations work when set to use a content addressable pool.
+    """
+    package = data.packages.joinpath("simple.dist-0.1-py2.py3-none-any.whl")
+    result = script.pip(
+        'install',
+        '--use-feature=content-addressable-pool',
+        '--content-addressable-pool-save-files',
+        '--cache-dir', './cache',
+        package,
+        allow_stderr_warning=True
+    )
+
+    # This is the sha of the simple.dist-0.1-py2.py3-none-any.whl wheel file
+    digest_name, b64hash = \
+        'sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU'.split('=', 1)
+    path = urlsafe_b64decode(f'{b64hash}=').hex()
+
+    # Make sure the path of the wheel we installed is safely installed in the cacache
+    exists = [
+        os.path.exists(
```

Comment:
can this also check that this pooled file is hardlinked to the installation directory? (same inode)

Reply:
For sure, i'll just need to find how to check that.

Reply:
you can call https://docs.python.org/3/library/os.html#os.lstat . then check the [`st_ino`](https://docs.python.org/3/library/os.html#os.stat_result.st_ino) on the result!

Reply:
I ended up going with `os.path.samefile`, which does that behind the scenes.